intel_extension_for_transformers/llm/runtime/graph/scripts/one_click_run.py

```diff
@@ -19,11 +19,16 @@
 from transformers import AutoConfig
 import subprocess
 
+build_path = Path(Path(__file__).parent.absolute(), "../build/")
+
 def main(args_in: Optional[List[str]] = None) -> None:
     parser = argparse.ArgumentParser(description="run quantization and inference")
     parser.add_argument(
         "model", type=Path, help="directory containing model file or model id"
     )
+    parser.add_argument(
```

Comment:
one click may not need this, but we should show logs when build folder is not exist

Reply:
we have logs 
```
    if not path.exists():
        print(path)
        print("Please build graph first or select the correct model name.")
```
one click script will call quantize.py and inference.py, so if user has user-defined build directory, user should call one-click script with build_dir path